new/LIT7056Qene.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7056Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለክርስቶስ፡ ጥዑመ፡ ቃል፡ ስቁል፡ ጴጥሮስ፡ በአጽምዖቱ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">La-Krǝstos ṭǝʿuma qāl sǝqul P̣eṭros ba-ʾaṣmǝʿotu (Qǝne of the śǝllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">When Peter listened to the sweet voice of the crucified Christ ...</title>
```

Comment:
ጥዑመ፡ ቃል፡ is an adjective, I believe, so I would translate: 

When Peter listened to the sweet-voiced crucified Christ…